src/main/java/core/basesyntax/exceptions/FileWriteException.java

```
@@ -0,0 +1,7 @@
+package core.basesyntax.exeptions;
+
+public class FileWriteException extends RuntimeException {
+    public FileWriteException(String message) {
+        super(message);
+    }
```

This exception class only defines a constructor with a message and does not provide a constructor that accepts a Throwable cause. The checklist requires: "Don't ignore exceptions in try-catch blocks... Let's rethrow a RuntimeException with an **informative** message and exception object." 

Because ReportExporterDaoImpl wraps IOExceptions into FileWriteException, you should add a constructor `public FileWriteException(String message, Throwable cause) { super(message, cause); }` and then rethrow with the original exception as the cause (for example: `throw new FileWriteException("Error writing report to " + fileName, e);`). This preserves the original stack trace and fulfills the checklist requirement.